examples/encoder_png.cc

```diff
@@ -118,14 +118,15 @@ bool PngEncoder::Encode(const struct heif_image_handle* handle,
     if (exifsize > 4) {
       uint32_t skip = (exifdata[0]<<24) | (exifdata[1]<<16) | (exifdata[2]<<8) | exifdata[3];
       skip += 4;
+      if (skip < exifsize) {
```

Comment:
Probably move the `if` before the `skip += 4` and write it as `skip < exifsize-4` to avoid overflows.

Reply:
Have updated to reflect this.